tests/core-replication-state.js

```diff
@@ -356,15 +362,15 @@ async function downloadCore(core, bits) {
 
 /**
  *
- * @param {CoreReplicationState} crs
+ * @param {CoreSyncState} crs
```

Comment:
same comment about updating the param name

```suggestion
 * @param {CoreSyncState} css
```